server/tests/controller/article/article.spec.js

```diff
@@ -27,8 +27,8 @@ const userDependencies = async userDetails => {
 const articleDependencies = async () => {
   const categoryInstance = await Categories.create(category);
   const categoryId = categoryInstance.get("id");
-  const articleInstance = Object.assign(articleSpec, { categoryId });
-  return Promise.resolve(articleInstance);
+  const articleInstance = Object.assign({}, articleSpec);
+  return Promise.resolve(Object.assign(articleInstance, { categoryId }));
```

Comment:
Why not spread this ?

Reply:
I have moved creating of articleSpec copy to the beforeEach block